package/doc/sphinx/source/documentation_pages/analysis/parallelization.rst

```diff
@@ -0,0 +1,306 @@
+.. -*- coding: utf-8 -*-
+
+=================
+Parallel analysis
+=================
+
+Starting from v2.8.0, MDAnalysis has introduced process-based parallelization for
+virtually all analysis classes. This page will briefly explain what has changed
+in the :class:`MDanalysis.analysis.base.AnalysisBase` protocol, how it affects
+users and developers, when you should use parallelization (almost always!), and
+when you should abstain from doing so (rarely).
+
+All this work was done as part of Google Summer of Code 2023 by @marinegor
+and MDAnalysis GSoC mentors.
+
+
+How to use parallelization
+==========================
+
+In order to use parallelization in a built-in analysis class ``SomeClass``, simply check which
+backends are available, and then just enable them by providing
+``backend='multiprocessing'`` and ``n_workers=...`` to ``SomeClass.run(...)`` method:
+
+.. code-block:: python
+
+    u = mda.Universe(...)
+    my_run = SomeClass(trajectory)
+    assert SomeClass.get_supported_backends() == ('serial', 'multiprocessing', 'dask')
+
+    my_run.run(backend='multiprocessing', n_workers=12)
+
+For some classes, such as :class:`MDAnalysis.analysis.rms.RMSF`,
```

Comment:
To be precise, RMSF can be parallelized with split-apply-combine---with a different algorithm (https://www.mdanalysis.org/pmda/api/rmsf.html). I would be more careful with the words here.

Reply:
Yes, indeed, my bad, I meant a different thing. I assume saying "parallelization of RMSF (in its current implementation) isn't possible" is correct, right?